tests/util/yamlWitnessStrip.ml

```diff
@@ -10,8 +10,13 @@ struct
 
   let strip_file_hashes {entry_type} =
     let stripped_file_hash = "$FILE_HASH" in
-    let location_strip_file_hash location: Location.t =
-      {location with file_hash = stripped_file_hash}
+    let location_strip_file_hash (location: Location.t): Location.t =
+      let file_hash =
+        match location.file_hash with
+        | Some _ -> Some stripped_file_hash (* TODO: or just map to None always? *)
```

Comment:
Was this addressed?

Reply:
Version 2.0 of the format makes some fields (like `file_hash` for every location) optional/unnecessary. So instead of changing all of them to `$FILE_HASH` in cram tests, we could just now omit them there. I'd prefer to do that in a separate PR though: it means all cram tests about witnesses get changed.